sdk/appcenter-distribute/src/main/res/values-es/appcenter_distribute.xml

```diff
@@ -12,10 +12,11 @@
   <string name="appcenter_distribute_downloading_mandatory_update">Descargando la actualización</string>
   <string name="appcenter_distribute_download_progress_number_format">%1$d MB de %2$d MB</string>
   <string name="appcenter_distribute_install_ready_title">Actualización de la aplicación descargada</string>
-  <string name="appcenter_distribute_install_ready_message">%1$s %2$s (%3$d) se ha descargado y está listo para instalarlo</string>
+  <string name="appcenter_distribute_install_ready_message">%1$s %2$s (%3$d) se ha descargado y está listo para instalarlo.</string>
```

Comment:
this is used for a system notification, notification messages usually end with dots, I just checked my phone with a system notification.